predictoor/pdr-parameters.md

```diff
@@ -0,0 +1,37 @@
+# Predictoor Parameters
+
+## Testnet Details
+As of Sep 12, 2023, Predictoor / Ocean contracts are deployed to [Oasis Sapphire testnet](https://docs.oasis.io/dapp/sapphire/#testnet).
+
+Users need fake OCEAN & ROSE tokens:  
+**- Fake OCEAN.** Staking & payment is in fake OCEAN. Users acquire it via a faucet. [Here's how](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/testnet-faucet.md).  
+**- Fake ROSE.** Gas fees are in fake ROSE. Users acquire it via a faucet. [Here's how](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/testnet-faucet.md).  
+
+## Mainnet Details
+As of Oct 10, 2023, Predictoor / Ocean contracts are deployed to [Oasis Sapphire mainnet](https://docs.oasis.io/dapp/sapphire/#mainnet).  
+
+Users need (real) OCEAN & ROSE tokens:  
+**- OCEAN.** Staking & payment is in OCEAN. [How to get OCEAN on Sapphire](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/get-ocean-on-sapphire.md).  
+**- ROSE.** Gas fees are in ROSE. [How to get ROSE on Sapphire](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/get-ocean-on-sapphire.md).  
+
+## Feeds Published
+For testnet, there are 10 feeds: X/USDT pair for each of the top-10 coins by market cap (ignoring stablecoins), 5m timescales, on Binance, >0% fees on Binance. Paid feeds. The coins are: X = BTC, ETH, BNB, XRP, ADA, DOGE, SOL, LTC, TRX, DOT  
```

Comment:
Remove "For testnet, there are 10 feeds: ..." paragraph.

Why: it's wrong, and it's redundant with the (correct) paragraph below.